ding_unilogin/js/ding_unilogin.js

```diff
@@ -13,25 +13,26 @@
    * really on, so we hack it in with JS.
    */
   Drupal.behaviors.ding_unilogin = {
-    attach: function (context, settings) {
+    attach: function (context) {
```

Comment:
If I look at https://www.drupal.org/docs/7/api/javascript-api/managing-javascript the syntax is:

`attach: function (context, settings) {`

So why remove the settings var here.... i known that javascript will not "go down"... but....

Reply:
Fixed in https://github.com/eReolen/ereolengo/pull/105/commits/d2f4244cb11710bf4833eb340732022cf0b1d976.